- Shaman build: [wip-perezjos-crimson-only-09-07-2026-PR69730](https://shaman.ceph.com/builds/ceph/wip-perezjos-crimson-only-09-07-2026-PR69730/88c7c261ff49149c2957dcf1539de205ee1ef889/debug/481875/)
- Teuthology run:
```bash
PR=wip-perezjos-crimson-only-09-07-2026-PR69730
$ teuthology-suite -s crimson-rados -p 101 --force-priority -c $PR  -f debug  --filter-out rgw  --filter seastore --suite-branch main --suite-repo https://github.com/ceph/ceph.git
2026-07-13 09:24:36,590.590 INFO:teuthology.suite.run:Test results viewable at 
```
https://pulpito-ng.ceph.com/jjperez-2026-07-13_09:19:53-crimson-rados-wip-perezjos-crimson-only-09-07-2026-PR69730-distro-debug-trial/
- As discussed via slack: 
 ```
teuthology-kill -o jjperez -m trial -r jjperez-2026-07-13_09:19:53-crimson-rados-wip-perezjos-crimson-only-09-07-2026-PR69730-distro-debug-trial
Traceback (most recent call last):
  File "/tank/home/jjperez/teuthology/virtualenv/bin/teuthology-kill", line 10, in <module>
    sys.exit(main())
             ^^^^^^
  File "/tank/home/jjperez/teuthology/scripts/kill.py", line 41, in main
    teuthology.kill.main(args)
  File "/tank/home/jjperez/teuthology/teuthology/kill.py", line 42, in main
    kill_run(
  File "/tank/home/jjperez/teuthology/teuthology/kill.py", line 52, in kill_run
    run_info = report.ResultsReporter().get_run(run_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tank/home/jjperez/teuthology/teuthology/report.py", line 389, in get_run
    response.raise_for_status()
  File "/tank/home/jjperez/teuthology/.venv/lib/python3.12/site-packages/requests/models.py", line 1028, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://paddles-paddles.apps.pok.os.sepia.ceph.com/runs/jjperez-2026-07-13_09:19:53-crimson-rados-wip-perezjos-crimson-only-09-07-2026-PR69730-distro-debug-trial/
```

--- a/src/crimson/os/seastore/seastore.cc
+++ b/src/crimson/os/seastore/seastore.cc
@@ -1866,9 +1866,6 @@ seastar::future<> SeaStore::Shard::do_transaction_no_callbacks(
     }
     add_latency_sample(op_type_t::DO_TRANSACTION, total);
   }
-  add_latency_sample(
-    op_type_t::DO_TRANSACTION,
-    seastar::lowres_clock::now() - ctx.begin_timestamp);
 
   add_onode_tree_sample(ctx.transaction->get_onode_tree_stats());
 

